### ISSUE TITLE
Synchronized fails to build with new Swift 3 compilers

### ISSUE THREAD

**Issue by cortices** [ide/Synchronized#5]
New swift compilers in Xcode 8 beta require the `SWIFT_VERSION` build setting.
### Compiler output

```
=== CLEAN TARGET Synchronized OF PROJECT Synchronized WITH THE DEFAULT CONFIGURATION (Release) ===

Check dependencies
“Use Legacy Swift Language Version” (SWIFT_VERSION) is required to be configured correctly for targets which use Swift. Use the [Edit > Convert > To Current Swift Syntax…] menu to choose a Swift version or use the Build Settings editor to configure the build setting directly.
```

Synchronized can be updated with this build setting set to the value for Swift 2.2 to build correctly in new compilers.

**Comment by ide** [comment]
A PR would be appreciated, I don't yet have Xcode 8 installed.

**Comment by ide** [comment]
Thanks for the PR. I published 2.0.2 to CocoaPods just now.
